composer.json

```diff
@@ -3,18 +3,19 @@
     "license": "MIT",
     "require": {
         "php": ">=5.5.9",
+        "graphp/graphviz": "0.2.*",
```

Comment:
For sure, it'll be fixed on next update. I just like and am used to order the dependencies, I find it more readeable.